contents/docs/web-analytics/web-vs-product-analytics.mdx

```diff
@@ -0,0 +1,225 @@
+
+  <ProductScreenshot
+    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/docs/product-analytics/trends/active-viewers-trend-light.png"
+    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload//posthog.com/contents/docs/product-analytics/trends/active-viewers-dark.png"
+    classes="rounded"
+    alt="Trend insight example"
+  />
+</details>
+
+<details>
+  <summary>Find where people are getting stuck during onboarding with <strong>funnels</strong></summary>
+
+  Use [funnels](/docs/product-analytics/funnels) to understand where people are getting stuck or dropping off in your flow.
+
+  <ProductScreenshot
+    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/docs/user-guides/funnels/identifying-drop-offs-light-mode.png"
+    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/docs/user-guides/funnels/identifying-drop-offs-dark-mode.png"
+    classes="rounded"
+    alt="Funnel insight example"
+  />
+</details>
+
+<details>
+  <summary>Follow users along their journey through your product with <strong>user paths</strong></summary>
+
+  Use [user paths](/docs/product-analytics/paths) to find which parts of your product people are actually using, where users are getting confused or stuck, etc.
+
+  <ProductScreenshot
+    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/docs/user-guides/paths/example-light-mode.png"
+    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/docs/user-guides/paths/example-dark-mode.png"
+    classes="rounded"
+    alt= "User path insight example"
+  />
+</details>
+
+<details>
+  <summary>Look for particularly engaging features with <strong>stickiness</strong></summary>
+
+  Use [stickiness](/docs/product-analytics/stickiness) to find engaging features by seeing how many times users perform an event within a specific time period.
+
+  <ProductScreenshot
+    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/v1716289464/posthog.com/contents/stickiness-light.png"
+    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/v1716289465/posthog.com/contents/stickiness-dark.png"
+    alt="Stickiness insight example"
+    classes="rounded"
+  />
+</details>
+
+<details>
+  <summary>Visualize user cohorts and <strong>retention</strong> metrics</summary>
+
+  Use [retention](/docs/product-analytics/retention) insights to find how many users sign up and come back to use your product after trying it.
+
+  <ProductScreenshot
+    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/retention_light_805120c74c.png"
+    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/retention_dark_c319006162.png"
+    alt="Retention insight example"
+    classes="rounded"
+  />
+</details>
+
+Check out the [product analytics docs](/docs/product-analytics) for more examples and guides.
```

Comment:
```suggestion
</details>

<br />

Check out the [product analytics docs](/docs/product-analytics) for more examples and guides.
```

needs a hard line break as it's a bit tight on preview

![Screenshot 2024-11-04 at 17 57 28](https://github.com/user-attachments/assets/0e2b9359-324d-42a1-bcab-888c17e7502b)